src/Components/Pages.tsx

```diff
@@ -42,6 +48,70 @@ function getServicesScene(routeMatch: OptionalRouteMatch) {
   });
 }
 
+function EmbeddedSceneWrapper(props: EmbeddedLogsExplorationProps) {
+  // Component is always null, doesn't look like we can embed something from the same app?
+  const { component: LogsDrilldownComponent, isLoading } = usePluginComponent<EmbeddedLogsExplorationProps>(
+    'grafana-lokiexplore-app/embedded-logs-exploration/v1'
+  );
+
+  // We don't want to re-render the entire app every time the props change, only once when the plugin component is done loading
+  // eslint-disable-next-line react-hooks/exhaustive-deps
+  const Component = useMemo(() => LogsDrilldownComponent, [isLoading]);
+  return isLoading || !Component ? <LoadingPlaceholder text={'Loading...'} /> : <Component {...props} />;
+}
+
+function getEmbedScene() {
```

Comment:
Nit.

```suggestion
function getEmbeddedScene() {
```

Reply:
Fixed in https://github.com/grafana/logs-drilldown/pull/1294/commits/0b898f1781e63ae26a09ea2b3a60ff7c355c11af